litestar/middleware/logging.py

```diff
@@ -332,34 +316,3 @@ def __post_init__(self) -> None:
 
         self.response_log_fields = tuple(self.response_log_fields)
         self.request_log_fields = tuple(self.request_log_fields)
-
-    @property
-    def middleware(self) -> DefineMiddleware:
```

Comment:
backwards compatibility; see earlier comment

Reply:
just before I continue on others @provinzkraut you are ok with this bit, specifically this or it's still breaking. Im asking because I entirely removed DefinedMiddleware:
from:
`def middleware(self) -> LoggingMiddleware:`
to:
`def middleware(self) -> DefineMiddleware:`

Reply:
Yeah, good enough IMO